<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1251</width>
      <height>681</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="realTimeTab">
     <attribute name="title">
      <string>Real Time Plots</string>
     </attribute>
     <widget class="QFrame" name="pyranometerFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>150</width>
        <height>260</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pyranometerStopButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Display</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="pyranometerLcd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pyranometerStartButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="pyranometerLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Solar Irradiance</string>
       </property>
      </widget>
      <widget class="QLabel" name="pyranometerUnit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>W/m2</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="lidarPlot" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>430</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="lidarFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>150</width>
        <height>260</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="lidarLcd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lidarLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Lidar distance</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lidarStartButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Display</string>
       </property>
      </widget>
      <widget class="QPushButton" name="lidarStopButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="lidarUnit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="pyranometerPlot" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>430</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="startRecordButton">
      <property name="geometry">
       <rect>
        <x>138</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Record all data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopRecordButton">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Record all data</string>
      </property>
     </widget>
     <widget class="QFrame" name="windSensorFrame">
      <property name="geometry">
       <rect>
        <x>609</x>
        <y>100</y>
        <width>211</width>
        <height>260</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="windSpeedLcd">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="windSensorLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Wind Sensor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="windStartButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>158</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Display</string>
       </property>
      </widget>
      <widget class="QPushButton" name="windStopButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="lidarUnit_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>m/s</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="windAngleLcd">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lidarUnit_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>°</string>
       </property>
      </widget>
      <widget class="QLabel" name="lidarUnit_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>120</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>° C</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="windTemperatureLcd">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>120</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="windSensorPlot" native="true">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>100</y>
        <width>331</width>
        <height>260</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>E</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="oplsFrame">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>380</y>
        <width>191</width>
        <height>260</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pyranometerStopButton_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop Display</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="methaneLcd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pyranometerStartButton_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Display</string>
       </property>
      </widget>
      <widget class="QLabel" name="oplsLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>OPLS</string>
       </property>
      </widget>
      <widget class="QLabel" name="oplsUnit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ppm</string>
       </property>
      </widget>
      <widget class="QLabel" name="oplsUnit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Methane</string>
       </property>
      </widget>
      <widget class="QLabel" name="oplsUnit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ethane</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="ethaneLcd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="oplsUnit_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ppm</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="oplsPlot" native="true">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>380</y>
        <width>430</width>
        <height>260</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="recordTab">
     <attribute name="title">
      <string>Recorded Plot</string>
     </attribute>
     <widget class="QFrame" name="recordFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>19</y>
        <width>151</width>
        <height>551</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="loadFileButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Load file ...</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>80</y>
         <width>131</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Select Data </string>
       </property>
       <widget class="QComboBox" name="recordDataSelect">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>40</y>
          <width>121</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Distance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wind Angle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wind Speed</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Temperature</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solar Irradiance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Methane concentration</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ethane concentration</string>
         </property>
        </item>
       </widget>
       <widget class="QLCDNumber" name="nbPoints">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>90</y>
          <width>31</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>pts</string>
        </property>
       </widget>
      </widget>
      <widget class="QToolButton" name="plotButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </widget>
     <widget class="QCustomPlot" name="recordPlot" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>1011</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
     <zorder>recordPlot</zorder>
     <zorder>recordFrame</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1267</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
